### ISSUE TITLE
Line breaking bug

### ISSUE THREAD

**Issue by steveruizok** [tldraw/tldraw#4804]
### To Reproduce

1. Visit [https://www.tldraw.com/s/v2_c_88xK-DMuxdSzoZpMbPKlY?d=v0.0.1920.1113.page](https://www.tldraw.com/s/v2_c_88xK-DMuxdSzoZpMbPKlY?d=v0.0.1920.1113.page) and fork the project
2. Edit the label, deleting the space after "research" and creating the space again

### Expected

The measured size of the text box should exactly reflect the measured size of the text input.

### Observed

The two measurements differ.

**Comment by mimecuvalo** [comment]
Ugh, annoying — I'll take a look.

**Comment by steveruizok** [comment]
This too https://www.tldraw.com/r/Py5PfKCr5RCsBzr87SpVR?d=v443.162.1920.1113.page

**Comment by GeorgVolkov** [comment]
maybe this solution will solve your problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by mimecuvalo** [comment]
I looked into this more and the root of this problem is actually more tied to the Draw font. The problem seems to be the slightly random variation in the font in some renders.

This is an issue we've seen before with this font, where it doesn't always consistently produce the same output.

**Comment by steveruizok** [comment]
Right, that makes sense. @ds300 you did some tweaks with the font a while back, think there's a way to reduce the variation like this? Otherwise I can also reach out to the author @arrowtype

**Comment by arrowtype** [comment]
The font is deliberately made to be a bit randomized. 

If you turn off the `calt` font feature, I believe it will prevent the randomization, and that may solve this. (It's possible it's using a different feature, and I'm forgetting which one. You can test it on a website like wakamai fondue.)

**Comment by mimecuvalo** [comment]
Thanks for writing in @arrowtype. It's appreciated - also the wakamai fondue site is new to me, very useful.
It looks like the `calt` feature doesn't disable randomization. For example `iii` still randomizes:
<img width="263" alt="Screenshot 2024-11-04 at 09 39 51" src="https://github.com/user-attachments/assets/5f359b08-6b01-4ea7-b402-f22c860e451d">

I recognize that this is feature of the font to be randomized (and maybe not something to turn off) so we're trying some different tactics to try to measure the text in a more consistent way on our end. 

If you have any other ideas though, lemme know!

**Comment by arrowtype** [comment]
Sorry, kept meaning to check on this while at a computer.

The randomization is controlled by the `rlig` feature. I’m not totally sure whether you can turn off `rlig` with `font-feature-settings` CSS. If not, you could perhaps subset the font to strip out that feature, and see what happens. Alternatively, and more easily, you could try using one of the "Normal" style woff2 fonts, which already have those alternate characters subset out, to limit file size. You can find them in the download at https://github.com/arrowtype/shantell-sans/releases

**Comment by mimecuvalo** [comment]
this will be addressed by @ds300 in https://github.com/tldraw/tldraw/pull/4895

**Comment by mimecuvalo** [comment]
Scratch the prev. comment — we didn't end up removing the randomization from the font file.
